cosi.proto

```diff
@@ -0,0 +1,183 @@
+syntax = "proto3";
+package cosi.v1alpha1;
+
+import "google/protobuf/descriptor.proto";
+
+option go_package = "cosi";
+
+extend google.protobuf.EnumOptions {
+    // Indicates that this enum is OPTIONAL and part of an experimental
+    // API that may be deprecated and eventually removed between minor
+    // releases.
+    bool alpha_enum = 1060;
+}
+
+extend google.protobuf.EnumValueOptions {
+    // Indicates that this enum value is OPTIONAL and part of an
+    // experimental API that may be deprecated and eventually removed
+    // between minor releases.
+    bool alpha_enum_value = 1060;
+}
+
+extend google.protobuf.FieldOptions {
+    // Indicates that a field MAY contain information that is sensitive
+    // and MUST be treated as such (e.g. not logged).
+    bool cosi_secret = 1059;
+
+    // Indicates that this field is OPTIONAL and part of an experimental
+    // API that may be deprecated and eventually removed between minor
+    // releases.
+    bool alpha_field = 1060;
+}
+
+extend google.protobuf.MessageOptions {
+    // Indicates that this message is OPTIONAL and part of an experimental
+    // API that may be deprecated and eventually removed between minor
+    // releases.
+    bool alpha_message = 1060;
+}
+
+extend google.protobuf.MethodOptions {
+    // Indicates that this method is OPTIONAL and part of an experimental
+    // API that may be deprecated and eventually removed between minor
+    // releases.
+    bool alpha_method = 1060;
+}
+
+extend google.protobuf.ServiceOptions {
+    // Indicates that this service is OPTIONAL and part of an experimental
+    // API that may be deprecated and eventually removed between minor
+    // releases.
+    bool alpha_service = 1060;
+}
+
+service Provisioner {
+    // This call is meant to retrieve the unique provisioner Identity.
+    // This identity will have to be set in BucketRequest.Provisioner field in order to invoke this specific provisioner.
+    rpc ProvisionerGetInfo (ProvisionerGetInfoRequest) returns (ProvisionerGetInfoResponse) {}
+
+    // This call is made to create the bucket in the backend.
+    // This call is idempotent
+    //    1. If a bucket that matches both name and parameters already exists, then OK (success) must be returned.
+    //    2. If a bucket by same name, but different parameters is provided, then the appropriate error code ALREADY_EXISTS must be returned.
+    rpc ProvisionerCreateBucket (ProvisionerCreateBucketRequest) returns (ProvisionerCreateBucketResponse) {}
+    // This call is made to delete the bucket in the backend.
+    // If the bucket has already been deleted, then no error should be returned.
+    // If the bucket has already been deleted, then no error should be returned.
```

Comment:
Line 65 and 66 are the same

Reply:
fixed, sorry

Reply:
Looks like you have not pushed your changes?  I still see two duplicate lines:).